src/Config/Stage/BuildInterface.php

```diff
@@ -14,4 +14,8 @@
  */
 interface BuildInterface extends StageConfigInterface
 {
+    /**
+     * The number of subdirectories that will be created to save the report
```

Comment:
it is not a number, it is a variable name to provide number ;)

Reply:
Changed